berkeley-mobile/Tab Bar/TabBarController.swift

```diff
@@ -37,6 +37,10 @@ class TabBarController: UITabBarController, UITabBarControllerDelegate {
         let attributes = [NSAttributedString.Key.font: Font.regular(11)]
         tabBarAppearance.setTitleTextAttributes(attributes, for: .normal)
         
+        if #available(iOS 15.0, *) {
+            tabBar.scrollEdgeAppearance = tabBar.standardAppearance
+        }
+        
```

Comment:
Is this needed? It seems the same as [line 33](https://github.com/asuc-octo/berkeley-mobile-ios/blob/73daeb2f312c93573999c792323a99fe135de7f3/berkeley-mobile/Tab%20Bar/TabBarController.swift#L33) above.